shared/constants/chat2/message.tsx

```diff
@@ -634,18 +633,15 @@ const uiMessageToSystemMessage = (
 ): Types.Message | null => {
   switch (body.systemType) {
     case RPCChatTypes.MessageSystemType.addedtoteam: {
-      // TODO @mikem admins is always empty?
-      const {adder = '', addee = '', team = '', admins = null} = body.addedtoteam || {}
+      const {adder = '', addee = '', team = ''} = body.addedtoteam || {}
       const roleEnum = body.addedtoteam ? body.addedtoteam.role : undefined
       const role = roleEnum ? TeamConstants.teamRoleByEnum[roleEnum] : 'none'
-      const isAdmin = (admins || []).includes(minimum.author)
```

Comment:
can't we just fetch the role of the author and keep this bit in?

Reply:
It doesn't do anything important, just changes one word.

Reply:
but most of those system messages have the `manage members` part, why not keep it?

Reply:
Just seemed like a waste of effort

Reply:
Note that it didnt even work before, since it only looked in the admin list, and not the owners